doc/guides/writing-tests.md

```diff
@@ -424,11 +424,15 @@ will depend on what is being tested if this is required or not.
 To generate a test coverage report, see the
 [Test Coverage section of the Building guide][].
 
+Coverage reports for the Node.js master branch are run nightly and can be 
+found [here][].
```

Comment:
Welcome, @srcmake, and thanks for the PR! This looks great to me, with the exception that I would very much like to avoid `here` as link text. Since the URL is concise, maybe it can be included inline:

```markdown
Nightly coverage reports for the Node.js master branch are available at
https://coverage.nodejs.org/.
```

Reply:
Thanks for the feedback @Trott! 

I've made the changes as per your request. (I wasn't sure how to format the link and had considered just adding it in-line directly, as you suggested.) (Also, the phrasing you used is much clearer and less verbose!)